src/components/isToAdressTheUser.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <p class='comment_user_text' v-if='isAddress'>
+        <span class='comment_user_text_first_word'>@{{ firstWord }}</span> {{ commentText.slice(firstSpaceIndex + 1) }}
+    </p>
+    <p className='comment_user_text' v-else>{{ commentText }}</p>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+
+export default {
+    props: {
```

Comment:
```suggestion
export default {
    name: 'YourComponentName',
    props: {
```
every component should specify it's name. fix here and everywhere else